data/colors_v2/mixins/light_mode.scss

```diff
@@ -95,40 +113,39 @@ $export: (
     muted: $pink-0,
   ),
 
-  // Mapping of v1 variables to v2 variables
+  // Primer
+  // This is only meant for Primer components
 
-  text: $text,
-  icon: $icon,
+  primer: (
```

Comment:
Can't really think of a good name for this group. Some alternatives:

- `misc`
- `internal`
- `components`